server/scripts/fetchdata/JsonCardSource.js

```diff
@@ -3,54 +3,54 @@ const fs = require('fs');
 const _ = require('underscore');
 
 class JsonCardSource {
-    getCards() {
-        let files = fs.readdirSync('thronesdb-json-data/pack');
-        let totalCards = [];
-        let types = JSON.parse(fs.readFileSync('thronesdb-json-data/types.json'));
-        let factions = JSON.parse(fs.readFileSync('thronesdb-json-data/factions.json'));
+    constructor() {
```

Comment:
Now that the JSON files are the only source for card data, there's an implication that the submodule must be updated every time we want to release new card implementations. Previously, it only needed to be updated if we were writing tests for a new card (since the tests do not use Mongo and need a deterministic data source).

If this is undesirable or we want more options, I could update this to take a directory path so that you can specify a version of the data repo other than what's in the submodule directory. That would probably be more flexible but also more error prone (need to remember to pull in the separate repo then run the script vs just pulling latest source here and running the script)

Reply:
That's fine, One Day(tm) there will be a build script that takes care of all that. In the mean time, I'll just put it on my list of things to remember